### DISCUSSION TITLE
[rejected] Mapowania identyfikatorów zamówień (stare formularze zakupowe)

### DISCUSSION THREAD

**Discussion by pgasidlo** [allegro/allegro-api#5094] [Suggestions - rejected]
Analizując rozliczenia w panelu Allegro natrafiam na "Prowizja od sprzedaży", oferta `10865403153`. Rozwijam.
Mam tam numer zamówienia: `889ac4e1-147b-11ec-8e1b-3387393f7d21`. Link nie działa - tj. nie ma takiego zamówienia.

Okazuje się, że jest to link do starego formularza zakupowego. Aktualny to `afbdb0a3-147b-11ec-9308-67f47e03fe44`.

Przydałoby się prezentować "aktualny" link do zamówienia - gwarantowało by to czytelność rozliczeń. Aktualnie płacę za coś i nie wiem co to jest.

Druga sprawa, to przydałby się zasób w API umożliwiający starego identyfikatora z aktualnym. To umożliwiłoby, w przypadku otrzymania 404 w pytaniu o stary formularz zakupowy, dowiedzieć się jaki jest aktualny.

**Comment by hahyes** [comment]
Nie wiem, czy to ten przypadek, ale może wystąpić sytuacja, gdzie pojedyncze zamówienia są łączone w ramach jednego koszyka. W takim wypadku stare identyfikatory są usuwane, a zostaje jeden połączonego zamówienia. Ale identyfikatory poszczególnych pozycji pozostają dokładnie takie same, więc masz punkt oparcia o rozpoznawanie połączeń.
